build.gradle.kts

```diff
@@ -51,7 +51,7 @@ allprojects {
 }
 
 group = "exchange.dydx.abacus"
-version = "1.7.71"
+version = "1.7.72"
```

Comment:
let's manually bump minor version instead, we made a breaking API change here (TradingRewards is consumed by clients directly)